InventoryCatalogApi/Model/DeleteSourceItemsBySkusInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryCatalogApi\Model;
+
+/**
+ * Delete source items by given product skus service.
+ *
+ * @api
+ */
+interface DeleteSourceItemsBySkusInterface
```

Comment:
I'm not sure that we should expose this service as an API. Current logic is private implementation to make source items consistent in case of sku change. Let's left only private implementation.

Reply:
@sidolov, Interface removed